hastexo/tasks.py

```diff
@@ -44,12 +45,15 @@ def run(self,
         heat = self.get_heat_client(configuration)
 
         # Launch the stack and wait for it to complete.
-        (status, error_msg, stack) = self.launch_stack(configuration,
-                                                       heat,
-                                                       stack_run,
-                                                       stack_name,
-                                                       stack_template,
-                                                       reset)
+        (status,
+         error_msg,
+         stack,
+         was_resumed) = self.launch_stack(configuration,
```

Comment:
Maybe you want to start working with named arguments here? This is getting a bit confusing.